app/controllers/spotlight/contact_email_controller.rb

```diff
@@ -0,0 +1,22 @@
+module Spotlight
+  ##
+  # CRUD actions for exhibit contact emails
+  class ContactEmailController < Spotlight::ApplicationController
+    rescue_from ActiveRecord::RecordNotFound, with: :record_not_found
```

Comment:
This is something Rails does by default. In `config.action_dispatch.rescue_responses`. You don't need to provide this here.

Reply:
Hmm, I see what you're doing here. This is for a custom JSON response. Rails doesn't seem to have a good way to customize that. 👍